src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -68,6 +117,10 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
         return `${args.options.period} is not a valid period type. The supported values are D7|D30|D90|D180`;
       }
 
+      if (args.options.outputFilePath && !fs.existsSync(path.dirname(args.options.outputFilePath))) {
+        return 'Specified outputFilePath where to save the file does not exist';
```

Comment:
This message is still confusing. The way I read it, is that the file path I specified doesn't exist, which could be perfectly fine when I want the report to be created in a new file. To make it clear, we could say something like `The specified path {/the/non/existent/dir/here} doesn't exist`

Reply:
Done. This is fixed.